projects/docs/docs/building-at-scale/caching.md

```diff
@@ -82,4 +115,4 @@ tuist cache print-hashes
 
 ### Unsupported configurations
```

Comment:
If we have issues describing these unsupported ones we should link them here, it may encourage others to try and fix them if they have more context.

Reply:
This is related to the old way of importing dependencies, not sure we can do much for it